Makefile

```diff
@@ -309,6 +312,9 @@ statics/bindata.go: .typescript ebpf.build $(shell find statics -type f \( ! -in
 	$(call VENDOR_RUN,${GO_BINDATA_GITHUB}) go-bindata ${GO_BINDATA_FLAGS} -nometadata -o statics/bindata.go -pkg=statics -ignore=bindata.go $(BINDATA_DIRS)
 	gofmt -w -s statics/bindata.go
 
+.PHONY: compile.debug
+compile.debug: debug.flags compile
```

Comment:
compile.debug rule is different from debug :
In case of compile.debug you bypass all the generating rules/files ( govendor genlocalfiles dpdk.build contrib )

So please keep on one way/entry point to build in debug
compile.debug will make confusion and for example make GO_BINDATA_FLAGS useless as genlocalfiles rule is not called
Please revert these changes

Reply:
ok